docs/troubleshooting/monitoring.md

```diff
@@ -316,3 +316,255 @@ You may encounter this when you install the Harvester v1.3.0 or higher version c
+kind: ClusterRole
+metadata:
+  annotations:
+    meta.helm.sh/release-name: rancher-monitoring-crd
+    meta.helm.sh/release-namespace: cattle-monitoring-system
+  creationTimestamp: "2023-01-09T11:04:33Z"
+  labels:
+    app: rancher-monitoring-crd-manager
+    app.kubernetes.io/managed-by: Helm
+  name: rancher-monitoring-crd-manager
+  ...
+rules:
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  verbs:
+  - create
+  - get
+  - patch
+  - delete
+
+$ kubectl get clusterrolebinding rancher-monitoring-crd-manager
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  annotations:
+    meta.helm.sh/release-name: rancher-monitoring-crd
+    meta.helm.sh/release-namespace: cattle-monitoring-system
+  creationTimestamp: "2023-01-09T11:04:33Z"
+  labels:
+    app: rancher-monitoring-crd-manager
+    app.kubernetes.io/managed-by: Helm
+  name: rancher-monitoring-crd-manager
+  ...
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: rancher-monitoring-crd-manager
+subjects:
+- kind: ServiceAccount
+  name: rancher-monitoring-crd-manager
+  namespace: cattle-monitoring-system
+
+$ kubectl get configmap -n cattle-monitoring-system rancher-monitoring-crd-manifest
+apiVersion: v1
+data:
+  crd-manifest.tgz.b64: ...
+kind: ConfigMap
+metadata:
+  annotations:
+    meta.helm.sh/release-name: rancher-monitoring-crd
+    meta.helm.sh/release-namespace: cattle-monitoring-system
+  creationTimestamp: "2023-01-09T11:04:33Z"
+  labels:
+    app.kubernetes.io/managed-by: Helm
+  name: rancher-monitoring-crd-manifest
+  namespace: cattle-monitoring-system
+  ...
+
+$ kubectl get ServiceAccount -n cattle-monitoring-system rancher-monitoring-crd-manager
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
+    meta.helm.sh/release-name: rancher-monitoring-crd
+    meta.helm.sh/release-namespace: cattle-monitoring-system
+  creationTimestamp: "2023-01-09T11:04:33Z"
+  labels:
+    app: rancher-monitoring-crd-manager
+    app.kubernetes.io/managed-by: Helm
+  name: rancher-monitoring-crd-manager
+  namespace: cattle-monitoring-system
+  ...
+```
+
+### Root Cause
+
+Those objects do not have the related labels/annotations required by the `ManagedChart` object, hence they are marked as `missging`.
+
+### Workaround
+
+To fix the issue, you may follow the steps below to re-create the `rancher-monitoring-crd` Managedchart object.
+
+1. Confirm the chart version of `rancher-monitoring-crd` from the `harvester-cluster-repo` service.
+
+    Get the `harvester-cluster-repo` service IP.
+
+    ```
+    $ kubectl get service -A | grep repo
+    cattle-system                     harvester-cluster-repo                        ClusterIP      10.53.146.66    <none>            80/TCP                         8d
+
+    The IP address is 10.53.146.66.
+    ```
+
+    Get the chart version from the `index.yaml` in the `harvester-cluster-repo`.
```

Comment:
It would be thoughtful if we kindly remind the user where to run the `curl` command (I assume it could be run on the Harvester node?)